pkg/render/logstorage.go

```diff
@@ -667,11 +667,10 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 					Command: []string{"/usr/bin/readiness-probe"},
 				},
 			},
-			// 30s (init) + 10 * 20s (timeout) + 9 * 5s (period) which is approximately 4 minutes
```

Comment:
I think you should leave and adjust this comment, it is showing the time that we're allowing for `account for a slow elasticsearch start`

Reply:
This comment is adjusted and added back.